python/lsst/analysis/tools/actions/plot/histPlot.py

```diff
@@ -66,6 +73,21 @@ class HistPanel(Config):
         "is plotted, the percentile limit is the maximum value across all input data.",
         default=98.0,
     )
+    expectedValue = Field[float](
```

Comment:
Perhaps a better generic name would be `referenceValue`?

Reply:
To put this into context somewhat, if I want to add a line to my RA histogram plot to show a particular RA of interest, I'm not sure that `expectedValue` would seem to best describe that line of reference.